rele/subscription.py

```diff
@@ -178,8 +180,29 @@ def sub_process_landscape_photos(data, **kwargs):
                       the sub regarding their attributes.
     :return: :class:`~rele.subscription.Subscription`
     """
+    if filter_by and not (
+        callable(filter_by)
+        or (
+            isinstance(filter_by, Iterable)
+            and all(callable(filter) for filter in filter_by)
+        )
+    ):
```

Comment:
What do think about placing this in the Subscription class. Im afraid that the sub decorator is not the only place that we can instantiate a Subscription. Now that we support class based subscriptions #168

Reply:
Moved.